db-schema/src/lib.rs

```diff
@@ -3,7 +3,7 @@
 /// Column families alias type
 pub type Col = &'static str;
 /// Total column number
-pub const COLUMNS: u32 = 16;
+pub const COLUMNS: u32 = 17;
```

Comment:
Adding a column requires a dummy migration to bump the database version.

@yangby-cryptape can you confirm it? Do we still need to migrate?